app.py

```diff
@@ -19,7 +19,13 @@ def strategy_1(x: int, y: int,
     :param att_dy: attacker y delta
     :return: dx, dy that the defender need to move
     """
-    dx, dy = -att_dx, -att_dy
+    new_x_att_pos = att_x - att_dx
+    new_y_att_pos = att_y - att_dy
+
+    dx = 0.05  * (new_x_att_pos - x)
+    dy = 0.05  * (new_y_att_pos - y)
```

Comment:
It would be better to keep the momentum vector of (dx, dy) equal to momentum vector of (att_dx, att_dy), aka: keep defenders moving at the same speed as attacker.